dla_cnn/data_model/DesiMock.py

```diff
@@ -0,0 +1,88 @@
+from astropy.io import fits
+import numpy as np
+
+
+class DesiMock:
+    """
+    a class to load all spectrum from a mock DESI data v9 fits file, each file contains 1186 spectra.
+    :attribute wavelength array-like, the wavelength of all spectrum (all spectrum share same wavelength array)
+    :attribute data, dict, using each spectra's id as its key and a dict of all data we need of this spectra as its value
+      its format like spectra_id: {'FLUX':flux,'ERROR':error,'z_qso':z_qso, 'RA':ra, 'DEC':dec, 'DLAS':dlas information} ,
+      dla information is a tuple, and its format is like (dla_id, z_qso, NHI)

```

Comment:
What if there is more than 1 DLA on the sightline?
Is it then a series of tuples?

Reply:
And actually, it is more than DLAs.  We are taking all systems with
NHI > 19.3.

Reply:
Yes, it will be a series of tuples. I have added some comments about it.